### ISSUE TITLE
Encoding problems

### ISSUE THREAD

**Issue by Articmar** [Tudmotu/gnome-shell-extension-clipboard-indicator#451]
When copied, the Cyrillic characters turn into this: 

![scrn](https://github.com/Tudmotu/gnome-shell-extension-clipboard-indicator/assets/118318866/229cae14-25e6-4246-b859-f0e1f3abc179)

### Software Information: 
- **Extension version:** 57
- **OS Name:** Fedora Linux 39 (Workstation Edition)
- **GNOME version:** 45.5
- **Window interface:** Wayland
- **Kernel version:** Linux 6.7.11-200.fc39.x86_64

**Comment by Tudmotu** [comment]
Duplicate of #414 .
Can you please share one of the strings you are trying to copy and the app from which you are copying it? (chrome, gedit, etc)

Thanks!

**Comment by Articmar** [comment]
> Duplicate of #414 . Can you please share one of the strings you are trying to copy and the app from which you are copying it? (chrome, gedit, etc)
> 
> Thanks!

The text on the screenshot was copied in LibreWolf (Flatpak). But it happens in different applications and not only in Flatpak. And I don't know what is the cause. 
For example the screenshot was taken after I copied the following text "Какие контейнеры лучше всего подходят для аудиокниг?" from the website: 
https://chat.lmsys.org/ 
But today I tried to copy the same text, from the same site, in the same LibreWolf (Flatpak), but the text copied normally. 
All I did since then was to reboot the PC.

**Comment by Tudmotu** [comment]
@Articmar it could be that you were using an older version. The extension only updates after a logout/login.
Closing for now but if it comes back, please feel free to reopen / open a new issue.

**Comment by Articmar** [comment]
> @Articmar it could be that you were using an older version. The extension only updates after a logout/login. Closing for now but if it comes back, please feel free to reopen / open a new issue.

No, the problem is not in the version, I used the latest version of the extension. 
I was talking about the fact that after rebooting, I failed to get the error by following the same steps. 
But I still keep seeing this error regardless of the application and/or the copied text.
